free-programming-books-ro.md

```diff
@@ -13,6 +13,11 @@
 * [Ajax](http://etutoriale.ro/articles/1483/1/Tutorial-Ajax/)
 
 
+### C++
+
+* [Noțiuni de programare în limbajul C++ - Adrian Runceanu](https://docplayer.es/65970256-Notiuni-de-programare-in-limbajul-c.html)
```

Comment:
Hey and welcome to free programming books!

Please put the author/vendor _after_ the page separate from the link.

```suggestion
* [Noțiuni de programare în limbajul C++](https://docplayer.es/65970256-Notiuni-de-programare-in-limbajul-c.html) - Adrian Runceanu
```